modules/openapi/component-protocol/scenarios/auto-test-plan-detail/components/executeInfo/render.go

```diff
@@ -310,3 +312,19 @@ func convertReportToConfig(env apistructs.PipelineReport) (apistructs.AutoTestAP
 
 	return config, nil
 }
+
+func getApiConfigName(env apistructs.PipelineReport) string {
+	if env.ID == 0 {
+		return ""
+	}
+	envByte, err := json.Marshal(env)
+	if err != nil {
+		return ""
+	}
+	configData := apistructs.PipelineReport{}
+	err = json.Unmarshal(envByte, &configData)
+	if err != nil {
+		return ""
+	}
+	return fmt.Sprintf("%v", configData.Meta[autotest.CmsCfgKeyDisplayName])
```

Comment:
how about `Meta` is nil.

Reply:
judge autotest.CmsCfgKeyDisplayName is existed in meta
add test if meta is nil